plugins/modules/ovirt_disk.py

```diff
@@ -552,8 +551,8 @@ def finalize_transfer(connection, module, transfer_id):
 
 
 def download_disk_image(connection, module):
-    transfers_service = connection.system_service().image_transfers_service()
-    hosts_service = connection.system_service().hosts_service()
+    connection.system_service().image_transfers_service()
+    connection.system_service().hosts_service()
```

Comment:
Actually I didn't go deep into code to check whether there are some "side effects" of the call.
Many times these methods are more than just a simple getter.
I.e., they open/initialize some service/connection and then return some service ID or handler or something. So I didn't delete the calls, since didn't want to introduce any bugs...

Reply:
Done.
Removed.